src/components/forms/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { createPaddingSpacing, getFormStyle } from '../../../utils/helpers';
+import { Icon } from '../../Icon';
+import { IconTypes, SSCIconNames } from '../../Icon/Icon.enums';
+import { Spinner } from '../../Spinner';
+import { Input } from '../Input';
+import SearchSuggestions from './SearchSuggestions';
+import { SearchBarProps } from './SearchBar.types';
+import { SearchSuggestionFormatsLabels } from './Search.enums';
+import useControlledInput from '../hooks/useControlledInput';
+
+const SearchBarWrapper = styled.div`
+  position: relative;
+`;
+
+const StyledInput = styled(Input)`
+  text-overflow: ellipsis;
+  ${createPaddingSpacing({ horizontal: 2.5 })};
```

Comment:
When I focus the search field the text move 1px to the right. It needs compensation because focus border have width 2px and default just 1px

Reply:
nice catch! fixed :)